envoy/api/v2/core/protocol.proto

```diff
@@ -33,6 +33,19 @@ message Http1ProtocolOptions {
   // envoy as their HTTP proxy. In Unix, for example, this is typically done by setting the
   // *http_proxy* environment variable.
   google.protobuf.BoolValue allow_absolute_url = 1;
+
+  // [#not-implemented-hide:]
+  // Handle incoming HTTP/1.0 and HTTP 0.9 requests.
```

Comment:
Nit: kill HTTP/0.9.

Reply:
Why?  We're handling it as of https://github.com/envoyproxy/envoy/pull/2792
Do you mean it should be formatted HTTP/0.9?

Reply:
Per my comment there, we shouldn't handle it at all.